Userland/Libraries/LibGUI/Action.h

```diff
@@ -24,6 +24,11 @@
 namespace GUI {
 
 namespace CommonActions {
+enum class QuitAltShortcut {
+    None,
+    CtrlW
```

Comment:
nit: include trailing `,` on all lines in the future

Reply:
Will do, thanks!